packages/lesswrong/components/recommendations/PostBottomRecommendations.tsx

```diff
@@ -117,7 +123,7 @@ const PostBottomRecommendations = ({post, hasTableOfContents, ssr = false, class
                 }
                 <AnalyticsContext pageSubSectionContext="moreFromAuthor">
                   {moreFromAuthorPosts?.map((post) => (
-                    <EAPostsItem key={post._id} post={post} />
+                    isEAForum ? <EAPostsItem key={post._id} post={post} /> : <PostsItem key={post._id} post={post} />
```

Comment:
```suggestion
                    <PostsItem key={post._id} post={post} />
```
`PostsItem` is just a wrapper that switches between `EAPostsItem` and `LWPostsItem` depending on the forum type